### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by mjpost**
I am simplifying the instructions for corrections.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/file-a-correction.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: File a Correction
+about: Correction paper data in the Anthology
```
Comment (mbollmann) [inline .github/ISSUE_TEMPLATE/file-a-correction.md:3]: Correct*ing*?

File: .github/ISSUE_TEMPLATE/file-a-correction.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+**Please be sure to add the Anthology ID in the title**
+
+[See here](https://www.aclweb.org/anthology/info/corrections/) to read about the three types of corrections.
+
+## Metadata correction: please describe the issue here**
+
+
+## Revision or erratum: please add the following information**
+
+- [ ] I have described attached the revised PDF or erratum to this issue
```
Comment (mbollmann) [inline .github/ISSUE_TEMPLATE/file-a-correction.md:19]: remove "described"

Comment (mjpost) [reply .github/ISSUE_TEMPLATE/file-a-correction.md]: Fixed.

**Comment by akoehn** [comment]
Not merging so @mjpost can squash-merge if wanted.

**Comment by mjpost** [comment]
Thank you!

**Pull request merged**